wasm/test_page/bergamot.html

```diff
@@ -38,16 +38,11 @@
 <div id="divtranslation">
     <label for="from">From</label>
     <textarea id="from" name="from">
-Una estrategia republicana para obstaculizar la reelecci�n de Obama
-Los dirigentes republicanos justificaron su pol�tica por la necesidad de luchar contra el fraude electoral.
-Ahora bien, el Centro Brennan considera esto �ltimo un mito y afirma que el fraude electoral es menos frecuente en los Estados Unidos que el n�mero de personas que mueren a causa de la ca�da de un rayo.
-De hecho, los abogados republicanos no han encontrado m�s que 300 casos de fraude electoral en los Estados Unidos en diez a�os.
-Una cosa es cierta: esas nuevas disposiciones afectar�n negativamente a la tasa de participaci�n.
-En ese sentido, estas medidas minar�n en parte el sistema democr�tico americano.
-Al contrario de lo que ocurre en Canad�, los estados americanos son responsables de la organizaci�n de las elecciones federales en los Estados Unidos.
-Y en esa misma l�nea una mayor�a de los gobiernos americanos promulgaron, a partir de 2009, nuevas leyes que dificultaban el proceso de inscripci�n o de votaci�n.
-Este fen�meno se ha extendido tras las elecciones de noviembre de 2010, que vieron el aumento de 675 nuevos representantes republicanos en 26 estados.
-En consecuencia, durante el a�o 2011 se introdujeron 180 proyectos de ley que restring�an el ejercicio del derecho de voto en 41 estados.
+Una estrategia republicana para obstaculizar la reelecci�n de Obama. Los dirigentes republicanos justificaron su pol�tica por la necesidad de luchar contra el fraude electoral.
```

Comment:
Are ? expected? I'm seeing something like this.
![image](https://user-images.githubusercontent.com/727292/108782274-336e2280-7563-11eb-8a5b-bdd03aa433cf.png)

Reply:
Looks like a unicode disagreement.